server/server.js

```diff
@@ -80,14 +79,9 @@ app.prepare().then(async () => {
     }
   });
 
-  router.post("/webhooks", bodyParser(), async (ctx) => {
-    const response = await Shopify.Webhooks.Registry.process({
-      headers: ctx.req.headers,
-      body: ctx.request.rawBody,
-    });
-
-    console.log(`Webhook processed with status code ${response.statusCode}`);
-    ctx.res.statusCode = response.statusCode;
+  router.post("/webhooks", async (ctx) => {
+    await Shopify.Webhooks.Registry.process(ctx.req, ctx.res);
+    console.log(`Webhook processed!`);
```

Comment:
Should we still output the status code here?

Reply:
This is essentially just an example, and if we get that far we'll be returning a 200 anyway, the middleware throws an error if something goes wrong. Which makes me wonder if it makes sense to catch errors here and make the logging more explicit. Seems like valuable documentation to have.

Reply:
Catching errors makes sense, as it is how one would expect the library be used. On the other hand, if this is just an example, it may be overkill.